.changeset/heavy-schools-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-forms': major
```

Comment:
major release mean breaking change. I don t see it, may be an error ?
If no error please document the breaking change